README.md

```diff
@@ -1,5 +1,15 @@
 # Terraform Enterprise Release Notes
 
-This repository contains the release notes for Terraform Enterprise and is updated automatically upon each release. More information about Terraform Enterprise can be found on [terraform.io](https://www.terraform.io/docs/enterprise/). 
+This repository contains the release notes for Terraform Enterprise and is updated automatically upon each release. More information about Terraform Enterprise can be found on [terraform.io](https://www.terraform.io/docs/enterprise/).
 
-Questions or Issues related to Terraform Enterprise can be directed to [support](https://support.hashicorp.com), your Technical Account Manager or your Sales Executive. 
\ No newline at end of file
+Questions or Issues related to Terraform Enterprise can be directed to [support](https://support.hashicorp.com), your Technical Account Manager or your Sales Executive.
+
+## Updating Release Notes (internal use only)
+
+Because of [the workflows in PTFE Releases][gh-ptfe-releases-ci]:lock: inability
+to overwrite files in this repository, updating release notes is a manual
+process. To update any release notes in this repository after they've been
+published, [submit a pull request][gh-new-pr] with the changes that need to be made.
+
+[gh-ptfe-releases-ci]: https://github.com/hashicorp/ptfe-releases/blob/master/.circleci/config.yml#L8-L21 "Config file for PTFE Releases"
+[gh-new-pr]: https://github.com/hashicorp/terraform-enterprise-release-notes/compare "Submit new PR in this repository"
```

Comment:
my only consideration is around the posting of the source repo in the public repo, but it's probably a moot point.

Reply:
hmm, so should I only reference it by name? that's a quick enough fix.

Reply:
yeah, I think that should be fine.